assets/javascripts/test_result.js

```diff
@@ -749,11 +749,36 @@ function renderInvestigationTab(response) {
         keyElement.style.verticalAlign = 'top';
         keyElement.appendChild(document.createTextNode(key));
 
+        var textLines = response[key].split('\n');
+        var textLinesRest;
+
+        if (textLines.length > 10) {
+            textLinesRest = textLines.slice(10, textLines.length);
+            textLines = textLines.slice(0, 10);
+        }
+
         var valueElement = document.createElement('td');
         var preElement = document.createElement('pre');
-        preElement.appendChild(document.createTextNode(response[key]));
+        preElement.appendChild(document.createTextNode(textLines.join('\n')));
         valueElement.appendChild(preElement);
 
+        if (textLinesRest) {
+            var preElementMore = document.createElement('pre');
+            preElementMore.appendChild(document.createTextNode(textLinesRest.join('\n')));
+            preElementMore.style = "display: none";
+
+            var moreLink = document.createElement('a');
+            moreLink.style = "cursor:pointer";
+            moreLink.innerHTML = "Show more";
+            moreLink.onclick = function(){
```

Comment:
I mean *all* occurrences. Not sure why you write code like this so often but you could fix this style issue easily with the following substitution: `s/([^\s(\[]){$/\1 {/g`